os/board/rtl8730e/src/component/wifi/wifi_interactive_mode.c

```diff
@@ -595,6 +595,7 @@ int8_t cmd_wifi_disconnect(void)
 void cmd_wifi_info(int argc, char **argv)
 {
 	int i = 0;
+	u8 *ifname[2] = {(unsigned char*)WLAN0_NAME, (unsigned char*)WLAN1_NAME};
```

Comment:
I think this needs to be reverted. Could you please check it once?

Reply:
yes, I am reverting this line.